graylog2-web-interface/src/views/stores/CurrentViewStateStore.ts

```diff
@@ -47,11 +48,13 @@ export const CurrentViewStateActions: CurrentViewStateActionsType = singletonAct
     titles: { asyncResult: true },
     widgets: { asyncResult: true },
     widgetPositions: { asyncResult: true },
+    focusWidget: { asyncResult: true },
   }),
 );
 
 type CurrentViewStateStoreState = {
   activeQuery: string;
+  focusedWidget: string | undefined | null,
```

Comment:
Do we need to put this in the `CurrentViewStateStore`? I would suggest creating a separate context for this instead, which provides the current state and the actions to modify it.

Reply:
@kmerz We could use the `SearchPageLayoutContext` for this.